### DISCUSSION TITLE
Keeping styling of inline tools after save

### DISCUSSION THREAD

**Discussion by DurhamSmith** [codex-team/editor.js#1921] [Q&A]
Currently using the marker tool as given in the tutorial https://editorjs.io/inline-tools-api
I am able to use the marker fine, but when I save the data and render it on a new page it no longer is hilighted. It's as if the tool was never used.

How to I solve this issue/persist data. 

I don't need a functioning marker tool. I am looking to render latex math with katex (inline) and then have that displayed on the new page.

**Comment by snettah** [comment]
I think you need to sanitize your data https://editorjs.io/inline-tool-sanitizing

**Comment by domingoladron** [comment]
@snettah, you're 100% right.  You'd imagine that would maybe go under a heading such as `Saving and Sanitizing your Data`, as yes, @DurhamSmith , this is totally key to persisting the data.  I bumped into this and had been scratching my head until I found your question, so cheers for saving me ;)

I will create a PR of the docs to get it a bit more clear.
